### ISSUE TITLE
minimum cashout

### ISSUE THREAD

**Issue by romeox6x66** [chipshotDEV/Faucet-Builder#2]
hello sir;
thank you for your script is very nice really i like xapo 

but sir i try to much to put minimum cashout 

and i try pay to someone to make it and he stolen the money and gone

please sir if you can make uptade with minimum cash out it will be very nice

i can pay for that no problem with me

**Comment by chipshotDEV** [comment]
What do you mean by minimum cashouts?

We are not making this for the money. In case we understand what you want and do it, we will leave it open source.

**Comment by goldkey0070** [comment]
What he means is to make a account box so non xapo users can cash out at 5500 instead of our xapo account filling up with ref payouts.

**Comment by Faucetrank** [comment]
yes same question as goldkey0070 how non xapo users are paid and what's the minimum payment for them ?

**Comment by Faucetrank** [comment]
and one more question what is this email destbogan@gmail.com in function.php ? how can i replace this email with mine ?
